docs/folders.md

```diff
@@ -25,7 +25,7 @@ Get a Folder's Items
 Folder items can be retrieved by calling the
 `FoldersManager.GetFolderItemsAsync(string id, int limit, int offset = 0, IEnumerable<string> fields = null, bool autoPaginate=false)`
 method. Use the `fields` option to specify the desired fields.
-Requesting information for only the fields you need can improve performance and reduce the size of the network request.
```

Comment:
TBH I'd clarify this even further. It can improve the performance of your application _by_ reducing the size of the network request.